dotCMS/src/integration-test/java/com/dotmarketing/filters/FiltersTest.java

```diff
@@ -601,7 +564,7 @@ public void forward(ServletRequest arg0, ServletResponse arg1)
     }
 
 
-    public void runTests() throws IOException, DotDataException {
+    public void runTests() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)